### DISCUSSION TITLE
API Request Returning 502 Persistently

### DISCUSSION THREAD

**Discussion by karinnainiguez** [community/community#24489] [Apps, API and Webhooks]
<p>Hello! </p>
<p>I’m using the GitHub API v3 with the Golang SDK.  I’m able to send the other requests in my program, but am having trouble with one call in particular.  Whenever I send a request to get all the repos in an organization, I am getting a 502 Server Error.  I am able to send other calls with that same client, so I don’t think that’s the issue.  However, I haven’t found any potential solutions to this in the documentation.  Any help woud be greatly appreciated!  Below is my function in case it helps: </p>
<pre><code>func getRepos(c *github.Client, org string) []*github.Repository {

	repos, _, err := c.Repositories.ListByOrg(
		context.Background(),
		org,
		&amp;github.RepositoryListByOrgOptions{
			Type: "public",
		},
	)

	if err != nil {
		fmt.Printf("Error getting Repos: %v", err)
	}
	fmt.Printf("Total repos: %v\n", len(repos))
	return repos
}
</code></pre>

**Comment by karinnainiguez** [comment]
<p>Just some more information, I’m able to send the same request through my browser and Postman, and get a successful response of the collection of Repos.  In my case, the org is “kubernetes”, and the request is: </p>
<p>GET <a href="https://api.github.com/orgs/kubernetes/repos" rel="nofollow noopener">https://api.github.com/orgs/kubernetes/repos</a></p>

**Comment by nikhita** [comment, marked as answer]
<p>You are getting bitten by <a href="https://github.com/google/go-github/issues/1037" rel="nofollow noopener">https://github.com/google/go-github/issues/1037</a>. Removing the Accept header values fixed the problem for me.</p>
